### PR TITLE
Suggest explanation: allow some personal data, in accordance to #676

### PR DESCRIPTION
**Pull request by mal-tee**

### CODE REVIEW

File: content/en/suggest.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ If you want to get into the details of how we collect the data for our company d
 Considering these tips when suggesting companies helps us a lot, but is <strong>completely optional</strong>. Whether you follow none of these tips, one, some, or all: We are happy about every suggestion!
 
 - The first and best starting point for obtaining information about a company is the privacy policy. It also makes for a great `source`.
-- We really don't want any personal data in our database. This includes names in emails, e.g. <code>john.doe@example.org</code>.
+- We only allow personal data in our database if its absolutely nesseccary. That means, we only accept e.g. <code>john.doe@example.org</code> as email if no generic privacy-related email like <code>privacy@example.org</code> exists.
```
Comment (baltpeter) [inline content/en/suggest.md:15]:
```suggestion
- We only allow personal data in our database if it's absolutely necessary. That means, we only accept e.g. <code>jane.doe@example.org</code> as the email if no generic privacy-related email like <code>privacy@example.org</code> exists.
```

File: content/de/suggest.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ Falls Du Dich etwas intensiver damit beschäftigen willst, was für Daten wir wi
 Wenn Du diese beim Vorschlagen berücksichtigst, erleichterst Du uns unsere Arbeit beim Bearbeiten Deiner Einreichungen. Die Tipps sind aber natürlich <strong>komplett optional</strong>. Egal ob Du keinen, einen, ein paar oder alle Tipps befolgst: Wir freuen uns über jeden Vorschlag!
 
 - Der erste und meist auch beste Anlaufpunkt um Daten über eine Firma zu finden ist die Datenschutzerklärung. Das ist auch direkt eine tolle Quelle.
-- Wir wollen wirklich keine persönlichen Daten in der Datenbank speichern, auch nicht, wenn eine (E-Mail)-Adresse einen Namen enthält, wie z.B. <code>kim.mustermensch@example.org</code>.
+- Wir wollen persönlichen Daten nur in der Datenbank speichern, wenn es wirklich nötig ist. Falls z.B. eine (E-Mail)-Adresse einen Namen enthält, wie <code>kim.mustermensch@example.org</code>, dann nehmen wir diese nur auf, wenn es keine generische Datenschutz-Adresse wie <code>datenschutz@example.org</code> gibt.
```
Comment (baltpeter) [inline content/de/suggest.md:15]:
```suggestion
- Wir wollen persönlichen Daten nur in der Datenbank speichern, wenn es wirklich nötig ist. Falls z.&nbsp;B. eine (E-Mail)-Adresse einen Namen enthält, wie <code>kim.mustermensch@example.org</code>, dann nehmen wir diese nur auf, wenn es keine generische Datenschutz-Adresse wie <code>datenschutz@example.org</code> gibt.
```

File: content/en/suggest.md (hunk 1 above)
Comment (mal-tee) [reply content/en/suggest.md]: > nesseccary  

wow :D

**Review by baltpeter — APPROVED** [review]
Thank you!

**Pull request merged**